recipes/traceback2/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{%set name = "traceback2" %}
+{%set version = "1.4.0" %}
+{%set hash_type = "sha256" %}
+{%set hash_val = "05acc67a09980c2ecfedd3423f7ae0104839eccb55fc645773e1caa0951c3030" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  {{ hash_type }}: {{ hash_val }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - pbr
+
+  run:
+    - python
+    - linecache2
+    - six
+
+test:
+  imports:
+    - traceback2
+
+about:
+  home: https://github.com/testing-cabal/traceback2
+  license: PSF 2.0
+  summary: 'Backports of the traceback module'
+
+extra:
+  recipe-maintainers:
+    - pmlandwehr
```

Comment:
Please feel free to add @anguslees here, @pmlandwehr, as he proposed such a recipe he was willing to maintain in PR ( https://github.com/conda-forge/staged-recipes/pull/537 ). If I'm wrong about this assumption, please let us know, @anguslees, and we can take you off after.